tests/ui/components/results/verticalresultscomponent.js

```diff
@@ -48,6 +61,29 @@ describe('vertical results component', () => {
     };
   });
 
+  it('the query propery is based on global storage when the component is created', () => {
+    const component = COMPONENT_MANAGER.create(VerticalResultsComponent.type, defaultConfig);
+    const query = component.query;
+
+    expect(query).toEqual('yext');
+  });
+
+  it('updates to global storage vertical results update the component results', () => {
+    const component = COMPONENT_MANAGER.create(VerticalResultsComponent.type, defaultConfig);
+
+    const verticalResults = {
+      searchState: SearchStates.SEARCH_COMPLETE,
+      resultsCount: 15,
+      results: 'yext search results',
+      verticalConfigId: 'verticalKey',
+      resultsContext: {}
+    };
+
+    component.core.globalStorage.set(StorageKeys.VERTICAL_RESULTS, verticalResults);
+
+    expect(component.results).toEqual('yext search results');
```

Comment:
would it be possible to update this test to expect the component html or state to be a certain way, instead of an internal variable?

Reply:
yes

Reply:
updating to get get this value from the state